README.md

```diff
@@ -10,36 +14,42 @@ provider/
 main.go: Executable entry-point.
 
 Defaults to run on localhost:9000.  Visit http://localhost:9000/api for OpenAPI definition of
-service.  Take a look at server/routes.go for a concise list of supported URLs.
-
-Build Instructions
-------------------
+service.  Take a look at `server/routes.go` for a concise list of supported URLs.
 
-These are temporary while waiting on some tegola PRs.  Things will be simpler before long.
+## Build Instructions
 
-go-wfs needs not-yet-merged features in the tegola branch `gpkg_autoconfig`.
+```bash
+# create directory for local env
+mkdir /path/to/golang-env
+export GOPATH=/path/to/golang-env
+# install tegola
+go get github.com/go-spatial/tegola
+# FIXME: temporary hack to check out gpkg_autoconfig branch
+cd $GOPATH/src/github.com/go-spatial/tegola
+git checkout gpkg_autoconfig
+go build
+# install go-wfs
+go get github.com/go-spatial/go-wfs
+```
 
-Here is how you can make that happen:
+## Running
 
-1. clone github.com/go-spatial/tegola
-1. cd tegola
-1. check out branch `gpkg_autoconfig`
-1. cd ..
-1. clone github.com/go-spatial/go-wfs
-1. cd go-wfs
-1. mkdir -p src/github.com/go-spatial/
-1. ln -s /<path>/<to>/tegola src/github.com/go-spatial/tegola
+```bash
+# start server on http://localhost:9000/
+go run main.go  # or go build main.go
+```
 
-Then run / build like you normally would for go:
 
-1. Make sure your GOPATH & GOROOT are set to /<path>/<to>/go-wfs & /<path>/<to>/<golang-installation>
-1. `go run main.go` or `go build main.go`
+## Requests Overview
 
-Overview
---------
 Features are identified by a _collection name_ and _feature primary key_ pair.
-To see the collections available hit /api/collectionNames.
-To see the feature pks available hit /api/featurePks.
-To see the data for a single feature, hit /api/feature.
-To see all the feature data for a collection hit /api/collection.
-To create a new, filtered, temporary collection hit /api/feature_set.
+
+- Collections: http://localhost:9000/api/collectionNames
+- Features pks: http://localhost:9000/api/featurePks
+- Single collection: http://localhost:9000/api/collection
```

Comment:
I think it's worth distinguishing between getting collection names & data of features in a collection in the endpoint descriptions.

Reply:
Fixed/done/rebased.